onnx/defs/tensor/defs.cc

```diff
@@ -2426,11 +2435,13 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Input(
             1,
             "grid",
-            "Input offset, 4-D tensor of shape (N, H_out, W_out, 2), "
-            "where H_out and W_out are the height and width of grid and output, "
-            "Grid specifies the sampling pixel locations normalized by the input spatial dimensions. "
-            "Therefore, it should have most values in the range of [-1, 1]. "
-            "If grid has values outside the range of [-1, 1], the corresponding outputs will be handled as defined by padding_mode.",
+            "Input offset of shape (N, D1_out, D2_out, …, Dr_out, r), where D1_out, D2_out, …, "
```

Comment:
I'm wondering about this character `…`. Should we replace it by three dots `...`?

Reply:
Good point. I will change it later.

Reply:
Fixed.